Alcatraz/Controllers/ATZPackageTableViewDelegate.m

```diff
@@ -85,23 +86,12 @@ - (NSView *)tableView:(NSTableView *)tableView viewForTableColumn:(NSTableColumn
     [view.descriptionField setStringValue:package.summary];
     [view.linkButton setImage:[self tableView:tableView websiteImageForTableColumn:tableColumn row:row]];
     [view.linkButton setTitle:[self tableView:tableView displayWebsiteForTableColumn:tableColumn row:row]];
-    [view.typeImageView setImage:[self tableView:tableView packageTypeImageForTableColumn:tableColumn row:row]];
     [view.installButton setTitle:([package isInstalled] ? @"REMOVE" : @"INSTALL")];
     BOOL hasImage = package.screenshotPath != nil;
     [view.previewButton setFullSize:hasImage];
     [view.previewButton setHidden:!hasImage];
-    if (package.screenshotPath) {
-        [view.previewButton setImage:[[self class] cachedImageForPackage:package]];
-        if (!view.previewButton.image) {
-            __block NSButton* imageButton = view.previewButton;
-            [[self class] fetchAndCacheImageForPackage:package progress:NULL completion:^(NSImage *image) {
-                imageButton.image = image;
-                [tableView noteHeightOfRowsWithIndexesChanged:[NSIndexSet indexSetWithIndex:row]];
-            }];
-        }
-    } else {
-        [view.previewButton setImage:nil];
-    }
+	view.screenshotPath = package.screenshotPath;
```

Comment:
Definitely should keep indentation consistent with the project; soft tabs, 4 spaces per tab.